gobblin-yarn/src/main/java/org/apache/gobblin/yarn/AbstractYarnAppSecurityManager.java

```diff
@@ -117,7 +118,11 @@ protected void startUp() throws Exception {
     this.loginExecutor.scheduleAtFixedRate(new Runnable() {
       @Override
       public void run() {
-        loginAndScheduleTokenRenewal();
+        try {
+          loginAndScheduleTokenRenewal();
+        }catch(Exception e){
+          LOGGER.error("Error login using keytab, will continue the thread and try next time.");
```

Comment:
Change the message to "Error during login, will continue the thread and try next time." Since this is super class and we may not use keytab to login

Reply:
Sure